frontend/storage/utils/apiRequestManager.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import { SERVER_ADRESS } from '../common';
+
+export async function apiRequestGet(endpoint: string, queryParams: { [key: string]: any }) {
```

Comment:
If you take a look at `EntityFactory` in the backend you'll see a type `Dictionary`. Would make the method signature a bit more readable, so here is my suggestion:
1. Create `common` directory in the root of the project (on the same level as `frontend` and `backend`)
2. Create `types.ts` file and put the `Dictionary` type there
3. Use it here and refactor EntityFactory to use that Dictionary